task3.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import "fmt"
+
+// Point indicates the coordinates of a point on a plane
+type Point struct {
+	x, y int
+}
+
+// Square contains the necessary data to describe the square
+type Square struct {
+	start Point
+	a     uint
+}
+
+// End сalculates the end coordinates of a square
+func (s Square) End() (uint, uint) {
```

Comment:
This method should return `Point` structure.

Reply:
Edited.